src/core_plugins/metrics/public/components/vis_types/timeseries/vis.js

```diff
@@ -80,8 +80,10 @@ function TimeseriesVisualization(props) {
   });
 
   const interval = series.reduce((currentInterval, item) => {
-    const seriesInterval = item.data[1][0] - item.data[0][0];
-    if (!currentInterval || seriesInterval < currentInterval) return seriesInterval;
+    if (item.data.length > 1) {
+      const seriesInterval = item.data[1][0] - item.data[0][0];
+      if (!currentInterval || seriesInterval < currentInterval) return seriesInterval;
+    }
```

Comment:
should we be worried at all that the folder is returning undefined as fallback iso the accumulator `currentInterval`? it did so before, so not sure.